Mutt crashes when connection that used SASL is closed 
and reopened again (e.g. network problem)
see http://bugs.debian.org/524420 and http://bugs.mutt.org/3206

--- a/mutt_sasl.c
+++ b/mutt_sasl.c
@@ -481,6 +481,7 @@
   conn->conn_close = sasldata->msasl_close;
   conn->conn_read = sasldata->msasl_read;
   conn->conn_write = sasldata->msasl_write;
+  conn->conn_poll = sasldata->msasl_poll;
 
   /* release sasl resources */
   sasl_dispose (&sasldata->saslconn);
